### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Earljohn25** [ChrisTitusTech/winutil#2597] [Q&A]
Is this ok please reply i use microwin?

![IMG20240821082228.jpg](https://github.com/user-attachments/assets/20d621f0-c2de-4c05-a294-4671124c0c6a)

![IMG20240821082233.jpg](https://github.com/user-attachments/assets/13a24513-0ab0-47cb-9f52-a9cd263f45d9)

**Comment by CodingWonders** [comment]
This is normal for permanent packages (like `FodMetadata` or the foundation package) or superseded packages.

Your image should still work correctly

**Reply by Earljohn25** [reply]
Thank you

**Reply by CodingWonders** [reply]
Please mark the comment as an answer to change the discussion status accordingly
